docs/book/src/multiversion-tutorial/testdata/project/Makefile

```diff
@@ -43,8 +43,8 @@ help: ## Display this help.
 
 .PHONY: manifests
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
-	# Note that the option maxDescLen=0 was added in the default scaffold in order to sort out the issue
-	# Too long: must have at most 262144 bytes. By using kubectl apply to create / update resources an annotation
+	# Note that the option maxDescLen=0 was added in the default scaffold to sort out the issue
+	# Too long: must have at most 262144 bytes. By using kubectl to create/update resources an annotation
```

Comment:
Could you please revert the command is kubectl apply that is why we have it as it is.